bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -13,7 +13,7 @@
   <h1 class="title-shadow-box">{{ _('Leadership') }}</h1>
 
   <p>
-  {{ _('Our leadership team is made up of both paid staff and volunteers who represent a breadth of experience and expertise in building products that people love and empowering communities to make a significant impact on the project and in the world.') }}
+  {{ _('Our leadership team is made up of both paid staff and volunteers with expertise building products and empowering communities.') }}
```

Comment:
CC @peiying2
This change landed without ever being flagged for l10n 👎 

Result: we have a half translated paragraph in all localized pages.